web_app_drawer/static/src/less/dropdowns.less

```diff
@@ -0,0 +1,55 @@
+/* Copyright 2016 LasLabs Inc.
+ * License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). */
+
+.dropdown-submenu {
```

Comment:
TIL about the four menu levels - that is one more than I am accounting for currently.

I'm just going to do something like a `menu-level-x` to push the menu in a certain amount, similar to the new BS4 [`m-t-x` classes](http://v4-alpha.getbootstrap.com/components/utilities/#spacing). Combine that with the `.dropdown-header` and I think we'll be golden